### DISCUSSION TITLE
Newest Tutorial Documentation Issue (Stopwatch)

### DISCUSSION THREAD

**Discussion by D8ATech** [Textualize/textual#975] [General]
I have been going through the textual documentation, specifically following step by step creating the stopwatch application.  I noticed that when I added the functionality of the start, stop, reset buttons that when I hit stop, then start again the clock started over at 0.

I compared line by line my written code to the finished example and I found the problem, so I looked back at the documentation and found the minor flaw.

In the image for stopwatch05.py in the def update_time(self) -> None: function the line is:
`self.time = monotonic() - self.start_time`

In the image for stopwatch06.py this same function doesn't indicate any code change based on the color change but in fact this function does change to:
`self.time = self.total + (monotonic() - self.start_time)`

**Comment by D8ATech** [comment]
This is based on the documentation in the gh-pages branch and the examples found on the css branch

**Comment by willmcgugan** [comment]
Good spot. Thanks. I'll fix that.

**Comment by D8ATech** [comment]
Fixed in the version I pulled today.  Thanks!
